index.html

```diff
@@ -37,7 +37,7 @@
     <h4 class="text-center mt-5">Handcrafted, home-made masterpieces</h4>
     <form class="form-inline mx-5" style="width:20rem ;">
       <input class="" type="search" placeholder="Enter your email address" aria-label="Search">
-      <button class="btn" type="submit">Get Started</button>
+      <button class="btn" type="submit">Let's Get Started</button>
     </form>
   </div>
```

Comment:
please kindly check the margins of this div

Reply:
- [ ] Kindly make your request specific and straight to the point. For instance you said'please kindly check the margins of this div' ''. Being more specific will help avoid review loops. :wink:

Reply:
- [OPTIONAL ] I put as optional because it was required in the previous review but not yet satisfied Please be as detailed and direct as possible with your request. For example, you requested that we "please gently examine the margins of this div." Enhancing specificity will aid in preventing review cycles. :wink: